content/en/docs/reference/scheduling/config.md

```diff
@@ -145,7 +145,11 @@ extension points:
   Extension points: `Score`.
 - `VolumeBinding`: Checks if the node has or if it can bind the requested
   {{< glossary_tooltip text="volumes" term_id="volume" >}}.
-  Extension points: `PreFilter`, `Filter`, `Reserve`, `PreBind`.
+  Extension points: `PreFilter`, `Filter`, `Reserve`, `PreBind`, `Score`.
+  {{< note >}}
+  `Score` extension point is enabled when `VolumeCapacityPriority` feature is enabled.
+  {{< /note >}}
+  extension point is enabled when `VolumeCapacityPriority` feature is enabled.
```

Comment:
Maybe you can also add a one line summary of what is prioritized, like "It prioritizes the smallest PVs that can fit the requested volume size"

Reply:
BTW: remove L152 - which is duplicated with L150.